breadcord/module.py

```diff
@@ -70,18 +70,17 @@ def load_settings_schema(self) -> None:
 
     def install_requirements(self) -> None:
         distributions = list(importlib.metadata.distributions())
-        missing_requirements = []
-        for requirement in self.manifest.requirements:
+        if missing_requirements := [
```

Comment:
guh???

I'm assuming this was an automated linter check, but while this could *technically* be considered better from a robot's perspective, I feel like the complexity of this seriously sacrifices code legibility. Is it worth it?